src/hooks/usePoolsHandler.tsx

```diff
@@ -1,38 +1,84 @@
 
-type PoolType = { asset: string; totalSupply: string; supplyApy: string; totalBorrowed: string; borrowApyVariable: string; borrowApyStable: string }
+type PoolType = {
+  asset: string
+  totalSupply: string
+  supplyApy: string
+  totalBorrowed: string
+  borrowApyVariable: string
+  borrowApyStable: string
+}
 
 export const usePoolsHandler = () => {
-  const [poolData, setPoolData] = useState<PoolType[]>()
+  const [poolData, setPoolData] = useState<PoolType[]>([])
 
   useEffect(() => {
-    setPoolData([
-      {
-        asset: "Binance",
-        totalSupply: "20,223,182,626",
-        supplyApy: "895",
-        totalBorrowed: "20,960,370",
-        borrowApyVariable: "1683",
-        borrowApyStable: "394",
-      },
-      {
-        asset: "Coinbase Exchange",
-        totalSupply: "2,815,007,914",
-        supplyApy: "945",
-        totalBorrowed: "1,756,438",
-        borrowApyVariable: "1274",
-        borrowApyStable: "5196",
-      },
-      {
-        asset: "Gate.io",
-        totalSupply: "1,652,717,489",
-        supplyApy: "549",
-        totalBorrowed: "2,658,869",
-        borrowApyVariable: "2565",
-        borrowApyStable: "1448",
-      },
-    ])
-  }, [])
+    if (poolData?.length === 0) {
+      let query = JSON.stringify({
+        query: `
+    {
+      poolCreateds {
+        id
+        pool
+        cfmm
+        protocolId
+        protocol
+        count
+      }
+    }
+    `,
+      })
+      callApi(query)
+        .then((res) => {
+          if (res?.poolCreateds?.length > 0) {
+            let data: PoolType[] = []
+            for (let i = 0; i < res.poolCreateds.length; i++) {
+              let obj: PoolType = {
+                supplyApy: '0',
+                asset: res.poolCreateds[i].pool,
+                totalSupply: res.poolCreateds[i].protocol,
+                totalBorrowed: res.poolCreateds[i].protocolId,
+                borrowApyVariable: res.poolCreateds[i].cfmm,
+                borrowApyStable: res.poolCreateds[i].count,
+              }
+              data.push(obj)
+            }
+            setPoolData(data)
+          }
+        })
+        .catch((err) => {
+          notifyError(err)
+        })
+    }
+    // setPoolData([
+    //   {
+    //     asset: 'Binance',
+    //     totalSupply: '20,223,182,626',
+    //     supplyApy: '895',
+    //     totalBorrowed: '20,960,370',
+    //     borrowApyVariable: '1683',
+    //     borrowApyStable: '394',
+    //   },
+    //   {
+    //     asset: 'Coinbase Exchange',
+    //     totalSupply: '2,815,007,914',
+    //     supplyApy: '945',
+    //     totalBorrowed: '1,756,438',
+    //     borrowApyVariable: '1274',
+    //     borrowApyStable: '5196',
+    //   },
+    //   {
+    //     asset: 'Gate.io',
+    //     totalSupply: '1,652,717,489',
+    //     supplyApy: '549',
+    //     totalBorrowed: '2,658,869',
+    //     borrowApyVariable: '2565',
+    //     borrowApyStable: '1448',
+    //   },
+    // ])
```

Comment:
Removed

Reply:
I'm wondering, it might be good to still have sample data and have a environment booleaan variable to switch fom sample data and subgraph data. The purpose for this is if developers do not have access to v1-strategies, they cannot create data on their own machines, also while developing the values on the page, we can kind of have a way to see how it will look once we have the values queried. Developers need to create stuff in their back end for subgraph to work. Can we put back the sample data @Pochetes

Reply:
Yeah, I didn't think of that. Keep it there

Reply:
Done